HACKING.md

```diff
@@ -1,5 +1,20 @@
 # Important notes on the code
 
+**You can generate an internal overview with doxygen. For this, use `make doc-doxygen` and you'll find an internal overview of all functions and symbols in `docs/html`.**
+
+# Comments
+
+- Comment system is held similar to JavaDoc
+    - Use `@param` to describe all input parameters
+    - Documentation comments should start with a double star (`/**`)
+    - Look forward to properly link all member types
+        - Functions with an appended `()`
```

Comment:
These 2 lines needs some more explanation, I can't make much sense of them.

Reply:
I've added an example and also the way to reference symbols/variables.